filters.go

```diff
@@ -122,19 +100,28 @@ func (client *Client) GetFilter(id ID) (*Filter, error) {
 	return &q.Account.Filter, HandleErrors(err, nil)
 }
 
-func (client *Client) ListFilters() ([]Filter, error) {
+func (client *Client) ListFilters(variables *PayloadVariables) (FilterConnection, error) {
 	var q struct {
 		Account struct {
 			Filters FilterConnection `graphql:"filters"`
```

Comment:
we are missing the extra arguments on the graphql query here

Reply:
done!